source/particle/world.cc

```diff
@@ -101,13 +101,22 @@ namespace aspect
       return *property_manager;
     }
 
+
```

Comment:
Add another line (3 empty lines is our default between functions in .cc files, 1 line in .h files).

Reply:
So what to do when there is a mix of 1, 2, and 3 empty lines in the .cc file (as in this case)? I shouldn't adjust them all in this PR right?